googletest/include/gtest/gtest.h

```diff
@@ -823,6 +831,7 @@ class GTEST_API_ TestInfo {
   const std::string test_suite_name_;  // test suite name
   const std::string name_;             // Test name
   const std::string tag_;              // Test tag
+  const char size_;                    // Test size
```

Comment:
Would it make sense to store `size` as an enum instead of a `char`? If we store as a char, we have to worry about casing (`S` vs `s`) and random, nonsensical input (like `Q`). If we use an enum, we can specify the available sizes and sidestep any issues with casing or random input.

Reply:
I didn't do enum so we could add more later. Using `gtest_size` you have to worry about upper and lower case. In the rtc framework I added `size` and it converts to upper case

Reply:
Gotcha - after looking through the PR in RTC I can see that the size tag is hidden from the user, so no problem here. I was thinking that users would have to manually input `S` or something.